src/libutil/util.cc

```diff
@@ -558,6 +558,18 @@ Path getDataDir()
     return dataDir ? *dataDir : getHome() + "/.local/share";
 }
 
+Strings editorFor(const Path & file, unsigned int line)
```

Comment:
extracted because it needed to be available from libexpr and didn't inherently depend on any nix expression stuff